OracleDatabase/SingleInstance/README.md

```diff
@@ -211,6 +219,34 @@ The Oracle Database inside the container also has Oracle Enterprise Manager Expr
     # Running the Oracle Database 21c XE image with the secret
     podman run -d --name=<container_name> --secret=oracle_pwd oracle/database:21.3.0-xe
 
+##### Encrypting database password (Supported from 23.4.0 onwards)
+
+Users can generate public-private key pair and pass database password (encrypted) and decryption (private) key to the container securely.
+
+* Generate public-private key pair.
+
+        openssl genrsa -out key.pem
+        openssl rsa -in key.pem -out key.pub -pubout
+
+* Create oracle password file.
+
+        echo "<Your Password>" > pwd.txt
+
+* Encrypt the password file using public key (key.pub)
+
+        openssl pkeyutl -in pwd.txt -out encrypted_pwd.txt -pubin -inkey key.pub -encrypt
+
+    Make sure to delete pwd.txt once done
```

Comment:
This could be more prescriptive and say "You should make sure you delete...".

Reply:
Fixed